services/web/server/src/simcore_service_webserver/studies_dispatcher/_core.py

```diff
@@ -0,0 +1,99 @@
+import uuid
+from copy import deepcopy
+from dataclasses import dataclass
+from functools import lru_cache
+from typing import Iterator, Optional, Tuple
+
+from aiohttp import web
+from pydantic import ValidationError
+
+MEGABYTES = 1e6
```

Comment:
well it's not really correct... that should be with 1024*1024*1024 bytes

Reply:
bits or bytes?

Reply:
well if I'm not mistaken... you wrote megabytes, so you might be right... and which by the way is called Mebibyte MiB [ref](https://en.wikipedia.org/wiki/Megabyte)

but yeah my bad it looks good :roll_eyes: it's a minus in line 70... right...